refactor: Lower camel case cleanup

mfederowicz opened the pull request:
### Summary

Cleanup some parts of code to meet one of the standards:

- lowerCamelCase names capitalize the first letter of each word, except the first which is always lowercase, even if it’s an acronym -  ([source](https://dart.dev/guides/language/effective-dart/style))

mfederowicz committed:
refactor: lowerCamelCase cases from dart analysis

mfederowicz committed:
refactor: lowerCamelCase cases from dart analysis

mfederowicz committed:
refactor: lowerCamelCase cases from dart analysis

refactor: lowerCamelCase cases from dart analysis and clean imports

lowerCamelCase refactor: cleanup exception variable format

mfederowicz committed:
Merge remote-tracking branch 'origin/alpha' into lowerCamelCase-cleanup

mfederowicz committed:
refactor: lowerCamelCase cases from dart analysis and clean imports

mfederowicz committed:
refactor: lowerCamelCase cases from dart analysis and clean imports

jorre127 commented on lib/Models/workout_data.dart:
```diff
@@ -72,13 +72,13 @@ class WorkoutData {
+      loadedWorkout.exercises = newExerciseList;
       //workout = loaded_workout;
-    } catch (Exception) {
+    } catch (e) {
```
Is there a reason you sometimes write exception and sometimes e? If not you should probably keep it consistent.

mfederowicz replied:
ok i replaced catch(e) to catch(exception)

mfederowicz committed:
lowerCamelCase refactor: cleanup exception variable format

mfederowicz committed:
lowerCamelCase refactor: cleanup exception variable format

dinurymomshad commented:
@tenshiAMD any idea why some fo the files are showing every line changed even though we change a single line or some portion of it? Might be related to that editor config?

mfederowicz commented:
@dinurymomshad afaik this problem is with changing endline encoding crlf(win) / cr(Macintosh ) / lf(Unix) - when i came to office iwill try to fix this - my dev env is all linux :)

mfederowicz committed:
lowerCamelCase refactor: revert endline encoding

mfederowicz committed:
lowerCamelCase refactor: revert endline encoding

mfederowicz committed:
lowerCamelCase refactor

tenshiAMD reviewed (changes requested):
@mfederowicz hi please revert first the changes in lib/Models/exercise.dart, we should only see lines that are affected not the whole file that has been changed.

mfederowicz commented:
@tenshiAMD maybe ill return to this topic after proper config editor, because Iam working in Android Studio on Linux, and after any changes in file with crlf it converts all file to selected format :(

mfederowicz committed:
refactor: lowerCamelCase-cleanup

mfederowicz committed:
refactor: revert line endings to crlf

mfederowicz committed:
refactor: revert line endings to crlf

mfederowicz committed:
refactor: revert line endings to crlf

mfederowicz committed:
refactor: revert line endings to crlf

mfederowicz commented:
ok @tenshiAMD i reverted changes in lib/Models/exercise.dart

one small tip, problem is with .editorconfig configuration:

```
end_of_line = lf
```
it forces Android Studio to change end of line from orginal to LF - in case of lib/Models/exercise.dart it was CRLF, thats why when I commited only changes in few lines, github thinks that I changed all lines in selected file

mfederowicz committed:
Merge branch 'lowerCamelCase-cleanup' of gh:mfederowicz/exer_log into lowerCamelCase-cleanup

tenshiAMD commented:
@mfederowicz looks good to me however there are a lot of commits, can you do cleanups? first, rebase to alpha branch then commit into one. After that, I think it is good to merge then we address the line endings after this. Thanks a lot! 👍

mfederowicz commented:
@tenshiAMD sorry for silly question but can you write me step by step ho to do those cleanups? I dont know what does it mean rebase to alpha branch especially in case forked repo :( of course I can make another clean branch from alpha and then apply changes from branch lowerCamelCase-cleanup :P

mfederowicz committed:
Merge branch 'alpha' into lowerCamelCase-cleanup

tenshiAMD commented:
> @tenshiAMD sorry for silly question but can you write me step by step ho to do those cleanups? I dont know what does it mean rebase to alpha branch especially in case forked repo :( of course I can make another clean branch from alpha and then apply changes from branch lowerCamelCase-cleanup :P

@mfederowicz you might need to force push your branch though. I think it is much easier if you create another branch and close this PR then make a new one.

mfederowicz committed:
refactor: revert line endings to crlf

mfederowicz committed:
Merge branch 'lowerCamelCase-cleanup' of gh:mfederowicz/exer_log into lowerCamelCase-cleanup

 Conflicts:
	lib/Models/exercise.dart

mfederowicz committed:
Merge branch 'lowerCamelCase-cleanup' of gh:mfederowicz/exer_log into lowerCamelCase-cleanup

 Conflicts:
	lib/Models/exercise.dart
